docs/azure_jumpstart_ag/contoso_supermarket/arc_defender_servers/_index.md

```diff
@@ -0,0 +1,94 @@
+
+- Run the following command to enable Microsoft Defender for Servers in your subscription:
+
+    ```shell
+    az security pricing create -n VirtualMachines --tier 'standard'
+    ```
+
+    ![Screenshot of enabling Microsoft Defender for Servers at subscription level](./img/02.png)
+
+    > **NOTE: Remember to follow the steps provided in the Cleanup section to disable Microsoft Defender for Servers, otherwise you may incur additional charges in your subscription.**
+
+## Azure Arc-enabled servers and Microsoft Defender for Servers Integration
+
+Now that you have successfully enabled Microsoft Defender for Servers, you will get recommendations to help you protect your resources, including your Azure Arc-enabled servers.
+
+Microsoft Defender for Servers will collect data from your Arc-enabled servers to monitor for security vulnerabilities and threats. The data collection will allow greater visibility into missing updates, non-secure OS settings, endpoint protection status, health and threat protection.
+
+- Navigate to Microsoft Defender for Cloud. In the **Inventory** section under **VM and Servers**, Defender will provide you with an overview of all the discovered security recommendations for your **Azure Arc-enabled servers**.
+
+    ![Screenshot of Microsoft Defender for Cloud Inventory](./img/03.png)
+
+    > **NOTE: it may take up to 30 minutes for your Azure Arc-enabled server to be shown in Microsoft Defender for Cloud Dashboard**
+
+- Select one of the Azure Arc-enabled servers. Microsoft Defender for Cloud will show the security recommendations:
+
+    ![Screenshot of Microsoft Defender for Cloud Recommendations](./img/04.png)
+
+- Select one of the recommendations. Each recommendation will include:
+  - A short description of what is being recommended.
+  - Related recommendations.
+  - The remediation steps to carry out in order to implement the recommendation. For specific recommendations, you may also get a **_Quick Fix_** that enables you to quickly remediate a recommendation on multiple resources.
+
+    ![Screenshot of Defender recommendation on Azure Arc-enabled server](./img/05.png)
+
+- For this Azure Arc-enabled server the recommendation _Machines should be configured to periodically check for missing system updates_ provides a **_Quick Fix_**. It is using an ARM template to enable a periodic update assessment on the Azure Arc-enabled server.
+
+    ![Screenshot of Defender recommendation Quick Fix ARM template](./img/06.png)
+
+- You can trigger the remediation by clicking on the **_Fix_** button:  
+
+    ![Screenshot of Defender recommendation Fix button](./img/07.png)
+
+- After you apply the recommendation, it will be now marked as healthy. Note that it can take several minutes after remediation completes to see the resources in the 'healthy resources' tab
```

Comment:
same as before. I had situations that the recommendation didn't become healthy and we opened a support ticket. Even here is difficult to provide troubleshoot steps since there are different recommendations which are remediated in different ways